required/firstaid/latex2e-first-aid-for-external-files.dtx

```diff
@@ -685,7 +685,7 @@
 %  We also need to redefine the internal label commands of cleveref.
 %  This must be done after cleveref, so we use an unique label.
 %    \begin{macrocode}
```

Comment:
This must be done after \pkg{cleveref} has made its changes in \texttt{begindocument} so we add it to the same hook using the hook label used by \pkg{cleveref}. This way it is guaranteed to overwrite the definitions.